parm/atm/variational/3dvar_drpcg.yaml.j2

```diff
@@ -29,8 +29,17 @@ cost function:
     state variables: [ua,va,t,DZ,delp,ps,sphum,ice_wat,liq_wat,o3mr,phis,
                       slmsk,sheleg,tsea,vtype,stype,vfrac,stc,smc,snwdph,
                       u_srf,v_srf,f10m]
-  background error: !INC ${BERROR_YAML}
-  observations: !INC ${OBS_LIST}
+  background error:
+{% filter indent(width=4) %}
```

Comment:
```suggestion
{% filter indent(width=2) %}
```

Reply:
See comment above

Reply:
This I agree should be indented wth 4.
I retract the comment on this particular line.  However for the hofx and lgetkf yamls, observations is not part of "cost function" and so the indent there is 2.